### PR TITLE
AlloCiné dataset card

### PR DESCRIPTION
**Pull request by mcmillanmajora**
Link to the card page: https://github.com/mcmillanmajora/datasets/blob/allocine_card/datasets/allocine/README.md

There wasn't as much information available for this dataset, so I'm wondering what's the best way to address open questions about the dataset. For example, where did the list of films that the dataset creator used come from?

I'm also wondering how best to go about talking about limitations when so little is known about the data.

### CODE REVIEW

File: datasets/allocine/README.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+[tf-allociné](https://huggingface.co/tblard/tf-allocine) achieves 97.44% accuracy on the test set. 
+
+## Known Limitations
+### Known social biases
+
+The social biases of this dataset have not yet been investigated.
+
+### Other known limitations
+
+The limitations of the Allociné dataset have not yet been investigated, however [Staliūnaitė and Bonfil (2017)](https://www.aclweb.org/anthology/W17-5410.pdf) detail linguistic phenomena that are generally present in sentiment analysis but difficult for models to accurately label, such as negation, adverbial modifiiers, and reviewer pragmatics.
```
Comment (sgugger) [inline datasets/allocine/README.md:117]:
```suggestion
The limitations of the Allociné dataset have not yet been investigated, however [Staliūnaitė and Bonfil (2017)](https://www.aclweb.org/anthology/W17-5410.pdf) detail linguistic phenomena that are generally present in sentiment analysis but difficult for models to accurately label, such as negation, adverbial modifiers, and reviewer pragmatics. 
```

File: datasets/allocine/README.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+The BCP-47 code for French is fr. Dialect information is unknown (see Speaker section for further details).
+
+## Who is/was involved in the dataset use and creation?
+### Who are the dataset curators?
+
+The Allociné dataset was collected by Théophile Blard. 
+
+### Who are the language producers (who wrote the text / created the base content)?
+
+The dataset contains movie reviews collected from Allociné.fr. The content of each review may include information and opinions about the film's actors, film crew, and plot.
```
Comment (sgugger) [inline datasets/allocine/README.md:54]: Maybe link to the website?

**Review by sgugger — APPROVED** [review]
Great work!

**Review by lhoestq — APPROVED** [review]
really cool :)

**Pull request merged**